### PR TITLE
actions: update daily version for testing

### PR DESCRIPTION
**Pull request by nashif**
Update versions.json with the daily commit that device testing will be
performed on.

### CODE REVIEW

**Review by galak — APPROVED** [review]

File: .github/workflows/daily_test_version.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# SPDX-License-Identifier: Apache-2.0
+
+name: Publish commit for daily testing
+
+on:
+  schedule:
+    - cron:  '50 22 * * *'
+  push:
+    tags:
+    - '*'
```
Comment (galak) [inline .github/workflows/daily_test_version.yml:11]: what does the '*' do?

**Comment by galak** [comment]
Seems like it would be nice to have a date/timestamp in the json ;).

File: .github/workflows/daily_test_version.yml (hunk 1 above)
Comment (nashif) [reply .github/workflows/daily_test_version.yml:11]: you tell me, I copied that from your west action :-)

**Comment by nashif** [comment]
> Seems like it would be nice to have a date/timestamp in the json ;).

yes, I just do not want to break people scripts with such a change. This will come soon though.

**Review by erwango — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**